src/styles/styles.js

```diff
@@ -1759,6 +1760,22 @@ const styles = {
         alignItems: 'center',
     },
 
+    pdfPasswordForm: {
+        nativeNarrowContainer: {
+            ...sizing.w100,
+        },
+        wideScreen: {
+            maxWidth: 350,
+        },
+        narrowScreen: {
+            ...spacing.p5,
+            ...sizing.h100,
+        },
+        confirmButton: {
+            ...spacing.pt4,
```

Comment:
Use styles.pt4 directly where `styles.confirmButton` is used. Similarly use the utility classes everywhere possible before creating new styles.